Dockerfile

```diff
@@ -1,9 +1,8 @@
 # Copyright (c) 2019 Zededa, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
-FROM lfedge/eve-alpine:6.2.0 AS build
-ENV BUILD_PKGS go git
-RUN eve-alpine-deploy.sh
+FROM golang:1.16.3-alpine3.13 AS build
```

Comment:
Why are you switching to this?

Reply:
With previous notation we have go 1.15.7-r0 (`(13/13) Installing go (1.15.7-r0)`) inside.